test/unittest/types/topic/TopicTest.cpp

```diff
@@ -167,7 +167,7 @@ TEST(TopicTest, stdout_operator)
     testing::internal::CaptureStdout();
     std::cout << dt;
     std::string dt_string = testing::internal::GetCapturedStdout();
-    ASSERT_EQ(dt_string, "Topic{topic1, type1}");
+    ASSERT_EQ(dt_string, "Topic{topic1;type1;0}");
```

Comment:
Please, use this PR to remove this test and use an string comparison instead.

Reply:
:+1: